### PR TITLE
Add `expo install` support

### PR DESCRIPTION
**Pull request by fson**
# Why

To make it easier to install unimodules and vendored native modules in managed apps where only a certain version (the one bundled in Expo Client) can be installed, I've added the `expo install` command to Expo CLI (see https://github.com/expo/expo-cli/pull/472).

To get version numbers, it consumes a file named `bundledNativeModules.json` in the `expo` package, which is added here.

I've also made the publish script and the scripts for updating vendored native modules update this file.

# How

Added the file, updated `publish-packages` task and the `updateVendoredNativeModule` function.

# Test Plan

Made sure running `gulp publish-packages --dry --force --forceVersions` or `gulp update-react-native-svg --allPlatforms` updates the file.

### CODE REVIEW

File: tools/gulpfile.js [hunk 1]
```diff
@@ -278,6 +288,7 @@ gulp.task('update-react-native-fbads', () => {
     targetAndroidPath: 'modules/api/fbads',
     sourceAndroidPackage: 'io.callstack.react.fbads',
     targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.fbads',
+    installableInManagedApps: false,
```
Comment (brentvatne) [inline tools/gulpfile.js]: is this not `true`?

File: tools/gulpfile.js [hunk 2]
```diff
@@ -232,6 +239,7 @@ gulp.task('update-react-native-admob', () => {
     targetAndroidPath: 'modules/api/components/admob',
     sourceAndroidPackage: 'com.sbugert.rnadmob',
     targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.admob',
+    installableInManagedApps: false,
```
Comment (brentvatne) [inline tools/gulpfile.js]: shouldn't this be true?

File: packages/expo/bundledNativeModules.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  "expo-sensors": "~4.0.0",
+  "expo-sharing": "~2.0.0",
+  "expo-sms": "~4.0.0",
+  "expo-speech": "~4.0.0",
+  "expo-sqlite": "~4.0.0",
+  "expo-task-manager": "~4.0.0",
+  "expo-web-browser": "~4.0.0",
+  "lottie-react-native": "2.5.0",
+  "react-native-branch": "2.2.5",
+  "react-native-gesture-handler": "^1.1.0",
```
Comment (tsapeta) [inline packages/expo/bundledNativeModules.json]:
```suggestion
  "react-native-gesture-handler": "~1.1.0",
```
I know it was just copied from `expo`'s `package.json` (we should fix it there), but our native deps shouldn't be tied with `^`. In terms of unimodules and kmagiera's libs, it's safe to use `~`.

File: tools/gulpfile.js (hunk 2 above)
Comment (fson) [reply tools/gulpfile.js]: There is `expo-ads-admob`, I assumed installing that package (as suggested by our docs) would be the way to use this module in Expo apps.

**Review by sjchmiela — APPROVED** [review]
👍

**Review by tsapeta — APPROVED** [review]
lgtm 👍

**Pull request merged**